JS/Clue.js

```diff
@@ -1,13 +1,52 @@
 class Clue {
+  constructor() {
+    this.question;
+    this.answerplace = 0; 
+    this.player = 0;
+    this.guessCount = 0;
+  }
+  checkAnswer(input) {
+    if( 'hello' != input && this.player === 3) {
+      this.player = 0;
+    } 
+    if ( round.currentClues[this.answerplace].answer !== input ) {
+     game.players[this.player].score  -= round.currentClues[this.answerplace].pointValue;
+     this.what++
+     if(this.guessCount === 3 ) {
+      alert("hello world yall all wrong")
+     }
+     domUpdates.displayScores()
+      this.player++
+    }
+    if( round.currentClues[this.answerplace].answer ===  input ) {
+      game.players[this.player].score  += round.currentClues[this.answerplace].pointValue;
+      domUpdates.displayScores()
+      domUpdates.tearDownCard()
+      domUpdates.rightAnswer()
+    } 
+  }
+
 
-  checkAnswer() {
-    //checks if player guess is right 
-    //if so run score multiplyer fn in round.js
-    //if not move to next player method in round.js
+  cluePopup(some) {
+    this.question = some.question;
+    this.answer = some.answer;
+    let div = $(
+      `<div class="popup">
+        <p class="clue-value">${some.question}</p>
+        <form class="clue-form">
+          <label> What is 
+            <input type="text" class="clue-input">?
+          </label>
+          <button class="clue-button">enter</button>
+        </form>
+      </div>`)
+    $('#board').append(div);
+    setTimeout(() => {
+    alert("time is up")
+  } , 10000)
```

Comment:
You will not be able to test this class method if there's jQuery in it. Move this logic into a method on `domUpdates`